docs/features/native_functions.md

```diff
@@ -0,0 +1,31 @@
+# Native Functions
+
+```js
+intInBase10(10) // Takes in an int and returns the string version.
+round(10.5) // Rounds a float to the nearest value.
+floor(10.5) // Floors a float to the nearest value.
+ceil(10.4) // Ceilings a float to the nearest value.
+charCode(\{a}) // Returns the code for a character.
+intCode(32) // Returns the chacter associated for an int.
+charAt(1, "abc") // Returns the character at an index of a string, takes in an int and a string and returns a maybe.
+substring(0, 2, "abc") // Takes in a start, an end, and a string and returns a maybe string.
+len("abc") // Takes in any value and returns the length of it.
+split(\{b}, "abc") // Splits a string based on a character.
+strip(" abc ") // Returns a string with removed whitespace from the end.
+range(0, 10, 1) // Returns a list of ints that takes in a start, and end, and a step value.
+type(10) // Takes in any type and returns the type of it as a string.
```

Comment:
Perhaps it should clarify that it takes in a value of any type rather than a type expression. Also, since `type` is a keyword, does this still work